charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,176 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+Kubernetes: false
+
+operator:
+  initcontainer:
+    repository: "docker"
+    tag: "20.10"
+  container:
+    repository: "kubesphere/fluent-operator"
+    tag: "v1.7.0"
+    # FluentBit operator resources. Usually user needn't to adjust these.
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  logPath:
+    crio: /var/log
+    containerd: /var/log
+
+fluentbit:
+  image:
+    repository: "kubesphere/fluent-bit"
+    tag: "v1.9.9"
+  # fluentbit resources. If you do want to specify resources, adjust them as necessary
+  # You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
+    # prometheus.io/scrape: "true"
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  volumes: []
```

Comment:
Do we need mounts for logs? It seems confusing to say no mounts but then we're tailing files - should this be "additional" mounts?

Reply:
Yes, here
https://github.com/fluent/helm-charts/pull/299/files#diff-76ae90b2c504f560452cc8baeed4ac3b9a53dc2eaae4164ccaec564207dfbfb9R35-R40

Reply:
Right, so this is extra volumes on top of the log volumes? I think it should be clarified and personally I would prefer the name `additional` or similar in there to make it obvious.

The values file should not require dipping into the templates to figure out what happens, at least at a high level.

Reply:
Agree, these're not log volumes. So `additionalVolumes` and `additionalVolumesMounts` looks good